day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,19 @@
+def speckled_frogs(num)
+
+  in_words = {0 => 'zero', 1 => 'one', 2 => 'two', 3 => 'three', 4 => 'four', 5 => 'five', 6 => 'six', 7 => 'seven', 8 => 'eight', 9 => 'nine', 10 => 'ten'}
+
+  until num == 0
```

Comment:
Arrays in ruby are 0 indexed, how could you accomplish the same thing with an array?